### PR TITLE
Backup/Restore of all Failsafe params, FC looptime change via configurator

### PR DESCRIPTION
**Pull request by tricopterY**
Trying to merge into development branch. I will delete my PR for master branch.

### CODE REVIEW

File: js/data_storage.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var CONFIGURATOR = {
-    'releaseDate': 1429014724688, // new Date().getTime() - Tue Apr 14 2015 13:32:17 GMT+0100 (BST)
+    'releaseDate': 1431142200587, // new Date().getTime() - Sat May 09 2015 04:30:08 GMT+0100 (BST)
```
Comment (hydra) [inline js/data_storage.js]: don't update this please.  If it's wrong when i prepeare for a release i will fix at that point.

File: tabs/configuration.html [hunk 2]
```diff
@@ -233,7 +233,16 @@
                 </div>
                 <span i18n="configurationBatteryMultiwiiCurrent"></span>
             </label>
-        </div>        
+        </div>
+        <!-- -->
```
Comment (hydra) [inline tabs/configuration.html]: no need for empty comment.

File: js/backup_restore.js [hunk 3]
```diff
@@ -417,6 +431,44 @@ function configuration_restore(callback) {
...
+                };
+            }
+            
+            if (configuration.FAILSAFE_RX_CONFIG == undefined) {
+                configuration.FAILSAFE_RX_CONFIG = {                    
+                    min_usec:               985,
+                    max_usec:               2115
+                };
+            }
+
```
Comment (hydra) [inline js/backup_restore.js]: seems to be quite a lot of code duplication there, can you fix it please?

**Comment by hydra** [comment]
Can you squash all the changes into a single commit please?

**Comment by tricopterY** [comment]
@hydra - I messed up again. I might as well pull the latest from Master and redo things in a new PR.
Thanks again for your sharp eyes.

**Comment by tricopterY** [comment]
Will do a new PR, if I can rebase/squash properly.

**Comment by tricopterY** [comment]
@hydra - please see #198

**Pull request closed without merging**